yarn-project/noir-libs/value-note/src/utils.nr

```diff
@@ -80,4 +181,75 @@ fn send_note(
         encryption_pub_key,
         note.serialise(),
     );
+}
+
+/*
+  Sends three amounts to three recipients.
+  Why three? Because one private call currently allows `MAX_NEW_COMMITMENTS_PER_CALL = 4` output commitments.
+  So we split the output notes as: 3 to recipients + 1 to the owner (the change note).
+*/
+fn send_notes(
+    context: &mut Context,
+    recipient_balances: [Set<ValueNote, VALUE_NOTE_LEN>; 3],
+    amounts: [Field; 3], 
+    recipients: [Field; 3],
+) {
+    // Creates (4 - 1 = 3) new notes for the recipients.
+    let mut recipient0_note = ValueNote::new(amounts[0], recipients[0]);
+    let mut recipient1_note = ValueNote::new(amounts[1], recipients[1]);
+    let mut recipient2_note = ValueNote::new(amounts[2], recipients[2]);
+
+    debug_log_format("o1: v = {0}, o = {1}", [recipient0_note.value, recipient0_note.owner]);
+    debug_log_format("o2: v = {0}, o = {1}", [recipient1_note.value, recipient1_note.owner]);
+    debug_log_format("o3: v = {0}, o = {1}", [recipient2_note.value, recipient2_note.owner]);
+
+    // Insert the new notes to the recipient's sets of notes.
+    recipient_balances[0].insert(context, &mut recipient0_note);
```

Comment:
do you think all the following could be done in a for loop? Or is there noir syntax issues?